app/components/primer/alpha/select_panel.rb

```diff
@@ -397,6 +376,7 @@ def initialize(
         dynamic_label_prefix: nil,
         dynamic_aria_label_prefix: nil,
         body_id: nil,
+        remote_form_arguments: false,
```

Comment:
I'm assuming this is mean to function like a feature flag, and will be removed at some point after verifying things are working smoothly? If that's the case, could we rename this to something like `use_experimental_remote_form` (or similar) so it's clear:

* this a boolean (there are similarly named `*_arguments` in PVC that accept a hash).
* this is a temporary prop

Reply:
Ya! I am okay with the name swap